### ISSUE TITLE
Create a simple neural network.

### ISSUE THREAD

**Issue by riley-rangel** [riley-rangel/learn-synaptic#1]
# Learn
Based upon: https://scrimba.com/casts/cast-1980

# Implementation
- [x] Install `synaptic` dependency.
- [x] Import `Layer` and `Network` from library.
- [x] Create three new`Layer`s with 2-3-1 neurons (input, hidden, output).
- [x] Connect the`Layer`s (input => hidden, hidden => output).
- [x] Create a new `Network`, which takes an object including the `Layer`s.
- [x] Define the desired learning rate. (eg. 0.3)
- [x] Activate the `network` and pass a `Array` of two values. (eg. [0, 0]
- [x] Backpropagate the `network` by supplying a learning rate and the expected value based upon the given array of values.
- [x] Test the `network` by supplying a new `Array` of values based upon the original values given.
